omaha/net/cup_ecdsa_pubkey.2.h

```diff
@@ -0,0 +1,31 @@
+// Copyright 2023 Google Inc.
```

Comment:
Maybe we could use our license for new files? cc @fmarier

Reply:
It depends where we took that file. If we wrote it ourselves we can choose the license we want, but if we took it from somewhere else, then we need to keep the existing copyright statement.

Reply:
The file was auto-generated from a tool in GitHub.com/Google/Omaha and came with this header.

Reply:
In that case, we should just leave the header as it is.